src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangesRestClientTest.java

```diff
@@ -54,16 +54,16 @@
         return Iterables.transform(Arrays.asList(
                 queryParameter(
                         new TestQueryRequest().withQuery("is:open")
-                ).expectUrl("/changes/?q=is:open"),
+                ).expectUrl("/changes/?q=is%3Aopen"),
```

Comment:
I have not tried yet, but I assume encoding the colon still works fine with the server?